lib/node_modules/@stdlib/number/float32/base/signbit/README.md

```diff
@@ -82,6 +82,95 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/number/float32/base/signbit.h"
+```
+
+#### stdlib_base_float32_signbit( x )
+
+Returns a `boolean` indicating if the sign bit for a [single-precision floating-point number][ieee754] is on (`true`) or off (`false`).
```

Comment:
This is not correct. See https://github.com/stdlib-js/stdlib/tree/6e32d9591ed2e6551de73ca0ff9ea2cc8f59824d/lib/node_modules/%40stdlib/number/float64/base/signbit#stdlib_base_float64_signbit-x-.

Reply:
We cannot rely on JS and C APIs being the same in terms of API signatures.